tools/deb/istio-discovery.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
copyright header missing

Reply:
It doesn't seem to be common in systemd either - but on my system I found 2 (rtkit and binfmt) with copyright. K8s doesn't seem to have it in their files. 
Do we have a policy on that ?